appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: '{branch}.{build}'
+shallow_clone: false
+platform: x64
+
+clone_folder: C:\gopath\src\github.com\DataDog\datadog-agent
+
+# environment must be set for python 64 bit
+environment:
+  GOPATH: C:\gopath
+  RUBY_PATH: C:\Ruby23-x64
+  RI_DEVKIT: C:\Ruby23-x64\DevKit\
```

Comment:
what's ruby for?

Reply:
The ruby devkit is where we're picking up gcc & friends.

Reply:
I don't think we should depend on this, let's merge but I want to discuss this more throughly

Reply:
Sure, but just for history; recall we couldn't install "winbuilds" on the appveyor machines.  So, given the available toolchains, this satisfied the gcc requirement.

I would be all for completely changing our toolchain to (among other things) not require winbuilds.  But that's beyond the scope of this PR.